assets/dev/scss/frontend/_container.scss

```diff
@@ -24,19 +24,18 @@
 	--margin-bottom: 0;
 	--margin-left: 0;
 	--padding-top: var( --container-default-padding-top, 10px );
-	--padding-right: var( --container-default-padding-right, 10px );
+	--padding-inline-end: var( --container-default-padding-right, 10px );
```

Comment:
Can we change this to `inline-end`? (and look for other files that will need to be updated as well)